example_configs/gitlab.md

```diff
@@ -0,0 +1,36 @@
+# GitLab Configuration
+
+## Placeholders
```

Comment:
I'm not sure we need this placeholder section. Maybe just a word that it's configured for users in the git_user group?